nodes/src/nodes/tool_http_request/http_client.py

```
@@ -179,8 +180,8 @@ def _build_response(resp: requests.Response, elapsed_ms: int) -> Dict[str, Any]:
     if 'json' in content_type or 'javascript' in content_type:
         try:
             parsed_json = resp.json()
-        except Exception:
-            pass
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)